go/adbc/driver/bigquery/connection.go

```diff
@@ -527,6 +529,14 @@ func (c *connectionImpl) newClient(ctx context.Context) error {
 					Msg:  fmt.Sprintf("The `%s` parameter is empty", OptionStringAuthRefreshToken),
 				}
 			}
+			if c.accessTokenEndpoint == "" {
+				c.accessTokenEndpoint = DefaultAccessTokenEndpoint
+				// Only use default server name if the access token endpoint is also set to default
```

Comment:
shouldn't the first condition then change to `c.accessTokenEndpoint == "" || c.accessTokenEndpoint == DefaultAccessTokenEndpoint`?

Reply:
That's reasonable, I don't think it's strictly necessary but should be fine.